docs/source/nlp/tutorials/quickstarts/plot_multi_label_classification.py

```diff
@@ -0,0 +1,228 @@
+# The benefit of using embedding on top of the properties is that they are able to detect semantic changes in the data.
+#
+# In our case, we see that are significant semantic differences between the train and test sets. Specifically,
+# we can see some distinct segments that distinctly contain more
+# samples from the train dataset or more sample from the test dataset. By hovering over the segments we can see
+# text samples contained in each one and find the common ground.
+#
+
+# %%
+# Model Evaluation
+# ----------------
+# The suite below is designed to be run after a model has been trained and requires model predictions which can be
+# supplied via the relevant arguments in the ``run`` function.
+
+train_preds, test_preds = just_dance_comment_analysis.\
+    load_precalculated_predictions(pred_format='predictions', as_train_test=True)
+train_probas, test_probas = just_dance_comment_analysis.\
+    load_precalculated_predictions(pred_format='probabilities', as_train_test=True)
+
+from deepchecks.nlp.suites import model_evaluation
+
+suite = model_evaluation(n_samples=1000)
+result = suite.run(train_ds, test_ds, train_predictions=train_preds, test_predictions=test_preds,
+                   train_probabilities=train_probas, test_probabilities=test_probas, model_classes=class_names)
+result.show()
+
+# %%
+# Model Eval #1: Train Test Performance
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# On the most superficial level, we can immediately see (in the "Did’t
+# Pass" tab) that there has been significant degradation in the Recall on
+# class “Pain and Discomfort”. Moreover, in seems there is a general deterioration in our model
+# performance on the test set compared to the train set. This can be explained
+# based on the data drift we saw in the previous suite.
+#
+
+# %%
+# Running Individual Checks
+# =========================
+#
+# Checks can also be run individually with relevant conditions. In this section,
+# we'll show you how to do that while showcasing one of our
+# most interesting checks - PropertySegmentPerformance.
+#
+
+from deepchecks.nlp.checks import PropertySegmentsPerformance
+
+check = PropertySegmentsPerformance(segment_minimum_size_ratio=0.05)
+check = check.add_condition_segments_relative_performance_greater_than(0.1)
+result = check.run(test_ds, probabilities=test_probas)
+result.show()
+
+# %%
+# In the display we can see some distinct property based segments that our model under performs on.
+#
+# By reviewing the results we can see that our model is performing poorly on samples that have a low level of
+# Subjectivity, by looking at the "Subjectivity vs Average Sentence Length" tab
+# We can see that the problem is even more severe on samples containing long sentences.
+#
+# In addition to the visual display, most checks also return a detailed data describing the results. This data can be
+# used for further analysis, create custom visualizations or to set custom conditions.
```

Comment:
I'd mention that it can be accessed via the result's .value...
or alternatively at least explicitly mention, this is how we can access a results value:
(and then it's clearer that the code does exactly that)

not critical, just can help make it a bit more accessible